src/supertux/menu/game_menu.cpp

```diff
@@ -50,13 +53,46 @@ GameMenu::menu_action(MenuItem* item)
       break;
 
     case MNID_RESETLEVEL:
-      MenuManager::instance().clear_menu_stack();
-      GameSession::current()->toggle_pause();
-      GameSession::current()->reset_button = true;
+      if(!g_config->confirmation_dialog)
+      {
+        // instantly reset level
+        MenuManager::instance().clear_menu_stack();
+        GameSession::current()->toggle_pause();
+        GameSession::current()->reset_button = true;
+      }
+      else
+      {
+        // reset confirmation dialog
+        std::unique_ptr<Dialog> dialog(new Dialog);
+        dialog->set_text(_("Are you sure?"));
```

Comment:
As I said, this should probably a uniform `#define` or `static` variable.

Reply:
Don't use #define use a const static string please.